policyengine_us/variables/gov/states/wv/tax/income/subtractions/low_income_earned_income_exclusion/wv_low_income_earned_income_exclusion.py

```diff
@@ -0,0 +1,29 @@
+from policyengine_us.model_api import *
+
+
+class wv_low_income_earned_income_exclusion(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "West Virginia low-income earned income exclusion"
+    defined_for = "wv_low_income_earned_income_exclusion_eligible"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://code.wvlegislature.gov/11-21-10/"
+        "https://tax.wv.gov/Documents/TaxForms/2021/it140.pdf#page=20"
+        "https://tax.wv.gov/Documents/PIT/2022/PersonalIncomeTaxFormsAndInstructions.2022.pdf#page=27"
+    )
+
+    def formula(tax_unit, period, parameters):
+        federal_agi = tax_unit("adjusted_gross_income", period)
+        person = tax_unit.members
+        wv_earned_income = person("earned_income", period)
```

Comment:
we also need a tax_unit.sum to sum up the vectorized values of the earned income